### ISSUE TITLE
Missing package emits wrong error message

### ISSUE THREAD

**Issue by chbndrhnns** [pawamoy/mkdocstrings#127]
**Describe the bug**

mkdocstring raises a `CollectionError` if it cannot import a module due to a missing dependency:

```
mkdocstrings.handlers.CollectionError: module 'mod' has no attribute 'code'
```

I feel this error message is misleading as there is an attribute to the module but there is an error with dependencies.

**To Reproduce**
1. Create a code snippet like this

```python
import aiohttp

class SomeClass:
    """
    comment
    """
```
2. Try to render the docs

```
# Reference

::: mod.code.SomeClass
```

**Expected behavior**
An error message that indicates the error in a better way.

**Screenshots**

**Information (please complete the following information):**
- OS: OS X
- `mkdocstrings` version: 0.12

**Comment by pawamoy** [comment]
Well, there's no way to know if the missing object is supposed to be a module, a class, an attribute, or else, but I guess we could change the message. What would you suggest? Something like this?

```
mkdocstrings.handlers.CollectionError: module 'mod' has no child 'code'
```

**Comment by chbndrhnns** [comment]
I missed that there is no difference between an internal package and a dependency. What you suggested would point in a better direction, I believe.

Would something like [this](https://stackoverflow.com/a/29751732/11236740) also be an option: 
If the package mentioned in the exception is found in the dependencies, print another error message?

**Comment by pawamoy** [comment]
Yes, good idea, we could parse the ImportError message, looking for the package name, and use [metadata](https://importlib-metadata.readthedocs.io/en/latest/using.html#distribution-requirements) to check if this package is a dependency.

**Comment by pawamoy** [comment]
This should be fixed with mkdocstrings 0.13.5 and pytkdocs 0.9.0, with which you can use the new path syntax and delimit modules from other objects with a colon: `package.module:MyClass.attribute1`. See https://pawamoy.github.io/pytkdocs/#details-on-new_path_syntax
